src/components/DocumentationTopic.vue

```diff
@@ -520,15 +539,37 @@ export default {
   }
 
   /deep/ .content + * {
-    margin-top: $stacked-margin-large;
+    margin-top: var(--spacing-stacked-margin-large);
   }
 }
 
-// remove border-top for first section of the page
+.minimized-description {
+  margin-bottom: 1.5em;
+}
+
 /deep/ {
   .no-primary-content {
+    // remove border-top for first section of the page
     --content-table-title-border-width: 0px;
   }
+
+  .minimized-content {
+    & > * {
+      margin-bottom: 1.5em;
+      margin-top: 1.5em;
+      --spacing-stacked-margin-large: 0.667em;
```

Comment:
I think you may want to move these property assignments so that they aren't applied to lots of different elements using the `*` selector as you have it here. We only need these to apply to some root parent element where we want everything within it to inherit these properties.

I think this would ideally either be directly within the `.minimized-content` class or possibly adding a minimal-only class to the entire root element of this component and defining them there.